src/scenes/Nuke/Nuke.tsx

```diff
@@ -17,16 +16,24 @@ function Nuke() {
   }, [handleLogout]);
 
   return (
-    <Page centered>
+    <StyledNuke>
       <BaseXL>Your local cache has been nuked</BaseXL>
       <Spacer height={32} />
       <GalleryLink to="/">
         <StyledButton text="Take me home" />
       </GalleryLink>
-    </Page>
+    </StyledNuke>
   );
 }
 
+const StyledNuke = styled.div`
```

Comment:
super nit - I see former `Page`s being styled with a few varying naming patterns now: `StyledHomePage`, `StyledNuke`, `StyledPage` - maybe we can use the same pattern for all of these to make it easy to quickly see which styled components are styled like pages?

Reply:
ooo yeah, great call! it almost makes me want to standardize this into a `Page` component that all views can use 😂 😂 😂